src/redux/categories/categoriesSlice.js

```diff
@@ -0,0 +1,20 @@
+import { createSlice } from '@reduxjs/toolkit';
+
+const categorySlice = createSlice({
+  name: 'category',
+  initialState: {
+    categories: [],
+    status: 'Under construction',
+  },
+  reducers: {
+    addCategory(state, action) {
+      state.categories.push(action.payload);
+    },
+    removeCategory(state, action) {
+      state.categories = state.categories.filter((category) => category !== action.payload);
+    },
```

Comment:
Nice job importing the `categoriesReducer` to the store as requested by the previous reviewer👍. But you still haven't implemented the second requirement from the previous reviewer😉. As pointed out by the previous reviewer, it is a requirement that the slice state of the categories should include a reducer that checks the status and always returns "under construction". kindly see my suggestions below. you can do well to implement it anyway so far it meets the requirement👍. note that it is the reducer that checks the status that should return "under construction"😉.

```suggestion
  name: 'category',
  initialState: {
    categories: [],
  },
  reducers: {
    checkStatus: (state, action) => {
      state.categories =
        action.payload === "Under construction"
          ? "Under construction"
          : state.categories
    },
```
,
```